From 77ff6b48ef027d05c6951e13167c0e72f0cf7f13 Mon Sep 17 00:00:00 2001
From: Patrick Rudolph <patrick.rudolph@9elements.com>
Date: Tue, 10 Oct 2023 11:29:32 +0200
Subject: [PATCH 04/13] configurations: sbp1: Update PHY LDOs

According to the KSZ9131RNX's datasheet the 1.2V PHY LDOs operate
on 1.25V with a non-uniform margin. Update the thresholds to prevent
a sensor warning, while the LDOs are within their normal operating
condition. Account for ADC noise and use slightly relaxed thresholds.

Tested: Confirmed on the webui that the PHY LDOs are no longer marked
        as critical.

Change-Id: Icfc65e4563223248b07984e52a6b80912c9c1eff
Signed-off-by: Patrick Rudolph <patrick.rudolph@9elements.com>
---
 configurations/sbp1_baseboard.json | 32 ++++--------------------------
 1 file changed, 4 insertions(+), 28 deletions(-)

diff --git a/configurations/sbp1_baseboard.json b/configurations/sbp1_baseboard.json
index 9bfd28e..93ebd14 100644
--- a/configurations/sbp1_baseboard.json
+++ b/configurations/sbp1_baseboard.json
@@ -241,25 +241,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 1.26
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 1.236
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower non critical",
-                    "Severity": 0,
-                    "Value": 1.164
+                    "Value": 1.34
                 },
                 {
                     "Direction": "less than",
                     "Name": "lower critical",
                     "Severity": 1,
-                    "Value": 1.14
+                    "Value": 1.2
                 }
             ],
             "Type": "ADC"
@@ -274,25 +262,13 @@
                     "Direction": "greater than",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 1.26
-                },
-                {
-                    "Direction": "greater than",
-                    "Name": "upper non critical",
-                    "Severity": 0,
-                    "Value": 1.236
-                },
-                {
-                    "Direction": "less than",
-                    "Name": "lower non critical",
-                    "Severity": 0,
-                    "Value": 1.164
+                    "Value": 1.34
                 },
                 {
                     "Direction": "less than",
                     "Name": "lower critical",
                     "Severity": 1,
-                    "Value": 1.14
+                    "Value": 1.2
                 }
             ],
             "Type": "ADC"
-- 
2.41.0

